installation/install.sh

```
@@ -4209,6 +4202,8 @@ unzip_el6_rpm="${ZSTACK_INSTALL_ROOT}/libs/unzip*el6*.rpm"
 
 if [ `uname -m` == "x86_64" ]; then
     zstore_bin="${ZSTACK_INSTALL_ROOT}/${CATALINA_ZSTACK_CLASSES}/ansible/imagestorebackupstorage/zstack-store.bin"
+elif [ x"${OS}" == x"OE2403" ]; then
+    zstore_bin="${ZSTACK_INSTALL_ROOT}/${CATALINA_ZSTACK_CLASSES}/ansible/imagestorebackupstorage/zstack-store.${uname -m}.abi2.bin"
```

Shellcheck (0.11.0)

[warning] uname is referenced but not assigned (for output from commands, use "$(uname ...)" ).

(SC2154)